vector/src/main/res/values/themes.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <!--TODO decide whether these themes are the right base-->
+    <style name="Theme.Vector.Light.Base" parent="@style/Theme.AppCompat.Light.NoActionBar" />
+    <style name="Theme.Vector.Dark.Base" parent="@style/Theme.AppCompat.NoActionBar"/>
+
+    <style name="Theme.Vector.Light.Common" parent="Theme.Vector.Light.Base">
```

Comment:
Unfortunately, I think that all the colors will need to be changed (since the dark theme 'inverts' all the colors). I'd like to make all the colors attributes anyway, so they can be changed easily in the future. 

I'm making a [script to help me do this though](https://gist.github.com/jgkamat/59f041d7cfe45cb2b0b757227680b9df), so I can keep re-applying the changes while avoiding merge conflicts.